packages/core/test/portal/portalTests.tsx

```diff
@@ -34,7 +34,29 @@ describe("<Portal>", () => {
         assert.lengthOf(document.getElementsByClassName(CLASS_TO_TEST), 1);
     });
 
-    it("propagates class names", () => {
+    it("invokes onChildrenMount after mounting", () => {
+        const onChildrenMount = sinon.spy();
+        portal = mount(
+            <Portal onChildrenMount={onChildrenMount}>
+                <p>test</p>
+            </Portal>,
+        );
+        assert.isTrue(onChildrenMount.calledOnce);
+    });
+
+    it("propagates portalClassName to detached container root", () => {
+        const CLASS_TO_TEST = "bp-test-klass";
```

Comment:
nit: don't really need the `k` spelling here since it's a string, but I guess it's fine haha